src/CachePlugin.php

```diff
@@ -338,12 +342,14 @@ private function configureOptions(OptionsResolver $resolver)
             'hash_algo' => 'sha1',
             'methods' => ['GET', 'HEAD'],
             'respect_response_cache_directives' => ['no-cache', 'private', 'max-age', 'no-store'],
+            'cache_key_generator' => null,
```

Comment:
Not sure if I should use null here or if I should instanciate the default generator. Using null will make sure we do not instanceiate unneeded objects.